lib/simulation/openfisca/mapping/index.js

```diff
@@ -76,7 +71,7 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
 
                         // Aggrégats
                         _.keys(periods).forEach(function (agregatKey) {
-                            if (agregatKey != 'anneeFiscalReference' && isPeriodWithin(agregatKey, moment(item.periode, 'YYYY-MM'))) {
+                            if (agregatKey != 'referenceFiscalYear' && isPeriodWithin(agregatKey, moment(item.periode, 'YYYY-MM'))) {
```

Comment:
No strong opinion on this, "fiscal year" makes sense in English, but maybe the meaning of "reference" isn't clear out of our domain.